gui/options.cpp

```diff
@@ -430,7 +430,7 @@ void OptionsDialog::build() {
 		if (ConfMan.hasKey("antialiasing", _domain)) {
 			_antiAliasPopUp->setSelectedTag(ConfMan.getInt("antialiasing", _domain));
 		} else {
-			_antiAliasPopUp->setSelectedTag(-1);
+			_antiAliasPopUp->setSelectedTag(0xffffffffu);
```

Comment:
For this kind of warning, I was advised long time ago to use uint(-1) (or equivalent)

Reply:
Changed this one